seed/models/analysis_messages.py

```diff
@@ -0,0 +1,31 @@
+from django.db import models
+
+from seed.models import (
+    Analysis,
+    AnalysisPropertyView
+)
+
+
+class AnalysisMessage(models.Model):
+    """
+    The AnalysisMessage represents user-facing messages of events that occur
+    during an analysis, like a breadcrumb trail.
+    """
+    DEFAULT = 1
+
+    MESSAGE_TYPES = (
+        (DEFAULT, 'default'),
+    )
```

Comment:
Sorry - missed this on my first pass. It's unclear what `default` stands for here as well. I do understand us not necessarily knowing what the different types should be yet, so I'm thinking this will just be an additional migration once we get to implementing these messages. No suggested change for me now on this one.

(We could modify the 0131 migration later, but that would mean we have to perform rollbacks on migrations in develop and locally.)